docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,118 @@
+---
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
+
+With Optimize, review heatmap displays for a closer look at the number of instances that took longer than average, based on duration distribution. As a result, reap actionable insights and rapidly identify the constraints of your system.
+
+See [What is Optimize](https://camunda.com/products/camunda-platform/optimize/reports/) for an in-depth overview of Optimize’s capabilities.
+
+[//]:# (The above is just a placeholder link.)
+
+## Set up 
+
+A Camunda Cloud user can launch Optimize from the Cloud Console — the interface where users create clusters, and launch both Operate and Tasklist. Therefore, ensure you’ve [created a Camunda Cloud account](./getting-started/create-camunda-cloud-account.md) before getting started with Optimize for SaaS users.
+
+So long as a user chooses a version of Zeebe when creating a cluster that includes Optimize, the user can access Optimize.  While creating a cluster, the creation modal will note if Optimize is included by each version of Zeebe that can be selected.
+
+Once you’ve created a cluster, take the following steps inside Cloud Console to access Optimize:
+
+1. Click the **Clusters** tab and select the cluster you’d like to analyze.
+2. Click the **Applications** tab.
+3. Select the **Optimize** box at the bottom of the page to launch Optimize.
+
+You can begin analyzing reports and dashboards with just two process versions. However, the more process versions you work with in Optimize, the more performance attributes and data trends you’ll be able to study. For the purposes of this guide, we’ve preconfigured several processes to demonstrate Optimize’s capabilities.
+
+## Create and analyze dashboards
+
+Within Optimize, **reports** are based on a *single* visualization, similar to a single chart or graph. **Dashboards** are aggregations of these visualizations, similar to a full spreadsheet of data collections, or a combination of several comparative charts and graphs. **Collections** are groups of these data sets, similar to project folders for organizational purposes where we can nest a series of dashboards and/or reports within.
+
+Once you open Optimize, you’ll first view the homepage for these collections, dashboards, and reports.
+
+To create a collection on the **Home** page, select **Create New > New Collection**. Name your collection and click **Create Collection**.
+
+Next, click on the collection to view its contents. You’ll notice you can again select Create New to draft new reports and dashboards.
+
+Let’s create a dashboard inside our first collection. Take the following steps:
```

Comment:
There's a risk that this documentation is deprecated on arrival based on an ongoing redesign. Talk to Asia about what the final user steps are going to be

Reply:
Met with Asia this morning and cleared up several sentences in the latest commit based on her review.